jx_sqlite/insert_table.py

```diff
@@ -300,6 +323,10 @@ def _flatten(data, uid, parent_id, order, full_path, nested_path, row=None, guid
                     _flatten(v, uid, parent_id, order, cname, nested_path, row=row)
                 elif c.type:
                     row[c.es_column] = v
+                    for r in rows:
+                        if r:
+                            insertion.rows.append(r)
+                    rows=[]
```

Comment:
This `for r in rows` loop should belong in the rest of the `elif len(c.nested_path) < len(nested_path):` block.